sdk/appcenter-auth/src/test/java/com/microsoft/appcenter/auth/AuthTest.java

```diff
@@ -149,6 +153,34 @@ private static void mockHttpCallSuccess(JSONObject jsonConfig, ServiceCallback s
         serviceCallback.onCallSucceeded(jsonConfig.toString(), headers);
     }
 
+    @Test
+    public void forwardMsalLogging() {
+        AppCenterLog.setLogLevel(VERBOSE);
+        Auth.getInstance();
+        String tag = "msalTag", expectedTag = LOG_TAG + TAG_DELIMITER + tag, message = "Message from MSAL";
+        com.microsoft.identity.common.internal.logging.Logger.verbose(tag, message);
```

Comment:
use an import

Reply:
Good catch, that was a copy/paste from the library source. They have 2 Logger classes in different packages, a public and an internal one that's probably why they did it this way. Anyways our tests only use the internal Logger so I'll use an import :).